tools/scripts/ansible/redact-db-playbook.yml

```diff
@@ -54,7 +54,8 @@
         group: cldradmin
         mode: '0640'
     - name: fetch and unzip db
-      command: bash /home/cldradmin/fetch-db.sh
```

Comment:
@btangmu https://ansible-lint.readthedocs.io/en/latest/default_rules.html#commands-should-not-change-things-if-nothing-needs-doing normally ansible (it seems) wants to only change state to a fixed status. I think our usage of this playbook is fine, but we need to quell the linter warning.